gpuflow/dataplane/gpu/cuda/cuda_lpm_factory.cu

```diff
@@ -31,6 +31,14 @@ __global__ void InitLPMTable(IPv4RuleEntry *ipv4_tbl_24) {
   ipv4_tbl_24[idx].external_flag = false;
 }
 
+__global__ void InitIPv6LPMTable(IPv6RuleEntry *ipv6_tbl_24) {
+  int idx = blockIdx.x * blockDim.x + threadIdx.x;
+  ipv6_tbl_24[idx].next_hop = 254;
+  ipv6_tbl_24[idx].valid_flag = false;
+  ipv6_tbl_24[idx].depth = 0;
+  ipv6_tbl_24[idx].external_flag = false;
+}
```

Comment:
Should still initialize `tbl8_ptr` in `nullptr` or `NULL`, 
```cuda
ipv6_tbl_24[idx].tbl8_ptr = nullptr
```

Reply:
ok, I would add it.